cnf-certification-test/networking/suite.go

```diff
@@ -155,7 +155,7 @@ func testUndeclaredContainerPortsUsage(env *provider.TestEnvironment) {
 		failedPod := false
 		for listeningPort := range listeningPorts {
 			if !declaredPorts[listeningPort] {
-				tnf.ClaimFilePrintf("%s is listening on port %d protocol %d, but that port was not declared in any container spec.", put, portInfo.PortNumber, portInfo.Protocol)
+				tnf.ClaimFilePrintf("%s is listening on port %d protocol %s, but that port was not declared in any container spec.", put, portInfo.PortNumber, portInfo.Protocol)
```

Comment:
So I finally found the reason for the same trace appearing [several times for the same port](https://github.com/test-network-function/cnf-certification-test/pull/560#issue-1391331519): this trace is wrongly printing `portInfo` instead of `listeningPort`. Can you fix that in this PR, please?
```
tnf.ClaimFilePrintf("%s is listening on port %d protocol %s, but that port was not declared in any container spec.", put, listeningPort.PortNumber, listeningPort.Protocol)
```
This won't make the TC to pass when istio is used, but at least the port will appear once, as it should be.

Reply:
Oh great!  Good catch.  I'll fix it.

Reply:
Fixed.